karma-story-editor.config.cjs

```diff
@@ -145,6 +174,8 @@ module.exports = function (config) {
       runInParent: true,
     },
 
+    crossOriginAttribute: false,
```

Comment:
> Description: When true, this will append the crossorigin attribute to generated script tags, which enables better error reporting for JavaScript files served from a different origin. Disable this when you need to load external scripts that are served without the necessary Access-Control-Allow-Origin header.